src/main/java/com/dabsquared/gitlabjenkins/listener/GitLabMergeRequestRunListener.java

```diff
@@ -79,7 +80,8 @@ private void addNoteOnMergeRequestIfNecessary(Run<?, ?> build, GitLabPushTrigger
     }
 
     private String getResultIcon(GitLabPushTrigger trigger, Result result) {
-        if (result == Result.SUCCESS) {
+        if (result == Result.SUCCESS
```

Comment:
The code that adds the notes to a MR will be moved to a `Publisher` like the `GitLabCommitStatusPublisher` (#201) so we should do the "Mark unstable builds as success in GitLab" stuff within this `Publisher` to prevent unnecessary job configuration migrations.